plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -419,7 +419,16 @@ public List<JdbcColumnHandle> getColumns(ConnectorSession session, JdbcTableHand
                             getInteger(resultSet, "DECIMAL_DIGITS"),
                             Optional.ofNullable(arrayColumnDimensions.get(columnName)),
                             Optional.empty());
-                    Optional<ColumnMapping> columnMapping = toColumnMapping(session, connection, typeHandle);
+                    Optional<ColumnMapping> columnMapping;
+                    try {
+                        columnMapping = toColumnMapping(session, connection, typeHandle);
+                    }
+                    catch (RuntimeException e) {
+                        // toColumnMapping connector implementation could be not full, broken or rely on broken jdbc driver
```

Comment:
Are you suggesting that postgresql driver is broken? I agree that in general jdbc could be broken somewhere, but in postgresql we should first find broken things and then try to ignore them.

Reply:
e.g. https://github.com/trinodb/trino/pull/15053#discussion_r1023906723
But it was not a problem till now..